src/seedu/addressbook/AddressBook.java

```diff
@@ -454,14 +459,25 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      */
     private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
-        for (String[] person : getAllPersonsInAddressBook()) {
+        searchForMatch(keywords, matchedPersons);
+        return matchedPersons;
+    }
+
+    /**
+     * Search for matched person based on keyword
```

Comment:
Here your `keywords` is a `Collection<String>`. It's useful to mention it, so the usage is clear.

And, minor thing, the comments of the rest of the codes uses 3rd person singular form like "Deletes".